docs/home/faq.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "FAQ"
+description: "Frequently asked questions from the Hyperbeam community"
+---
+
+## Proxies
+<AccordionGroup>
+	<Accordion title="Is there an IP range I can use to whitelist Hyperbeam servers?">
+    Not at the moment. Hyperbeam uses a variety of server providers which makes it difficult to whitelist Hyperbeam servers based on IP. You can use username/password authentication to ensure traffic originates from Hyperbeam.
+	</Accordion>
+	<Accordion title="Where do I put the credentials in FindProxyForURL?">
+    ```json
+      {
+        "http_proxy": {
+          "pac": "function FindProxyForURL(url, host) {
+            return {‘PROXY 255.255.255.255:12345'}”,
```

Comment:
this is not valid json (there is a newline)

Reply:
to add to amby's comment, it's seems you're using "fancy quotations"; you'll want to swap that out for simple ASCII quotes

Reply:
Great catches. Will fix.